lib/facility.rb

```diff
@@ -1,14 +1,188 @@
+# Helper module to title case strings.
+module Helper
```

Comment:
Yeah, I like that you used a module for these helper methods. We haven't learned this yet, but a module would get it's own file and then we'd use `include Helper` in the classes we want to "mixin" this code. We would test the Module by testing the methods in the spec file of the class we're using them in. So if you use these methods in the Facility class, you'd write tests for them in the facility_spec.rb.